dumper/app/build.gradle

```diff
@@ -113,12 +115,12 @@ task cloudExtractorStartScripts(type: CreateStartScripts) {
 licenseReport {
     // onlyIf { ! gradle.startParameter.offline }
     filters = [
-        new com.github.jk1.license.filter.LicenseBundleNormalizer(bundlePath: rootProject.file("gradle/license-bundle-normalizer.json"), createDefaultTransformationRules: true)
+            new com.github.jk1.license.filter.LicenseBundleNormalizer(bundlePath: rootProject.file("gradle/license-bundle-normalizer.json"), createDefaultTransformationRules: true)
```

Comment:
It looks from the PR as if this whitespace change changed both the spaces-to-tabs (I assume) but also the indentation level, which is now apparently inconsistent (4 vs 8). Is that so?

Reply:
That was intellijs doing. As far as I can tell, intellij applies groovy rules and uses the continuation indentation rules for list elements. After setting this to 4, it formatted it as it was before. I also increased column width to 120 to avoid the line breaks it introduced.